docs/src/pages/index.soy

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Home"
-description: "A collection of well known Algebraic Data Types for your utter enjoyment."
```

Comment:
Why this change? This is the tag line for the lib.

Reply:
I'll revert this change.  It looked nice on paper